### ISSUE TITLE
cursor does not change

### ISSUE THREAD

**Issue by gdkrmr** [gabesoft/evil-mc#79]
The following piece of text:
```latex
\begin{itemize}
\item item 1
\end{itemize}
```
1) in normal mode move the cursor to the first `itemize`
2) `C-n` to place second cursor on second itemize
3) `ci}` to change the environment type

everything works fine except the cosmetic issue that the cursor just does not change to the insert mode cursor.

**Comment by gabesoft** [comment]
There is a variable called `evil-mc-enable-bar-cursor` that controls whether the cursor should be change to a bar when in insert mode. That should be enabled by default though. Also, just to note that bar cursors only look good on Linux. On mac it is simulated with `|` character but that results in shifting all text to the right by one character. Myself, I only have it enabled on Linux.

**Comment by gdkrmr** [comment]
I am on linux and doing `C-n` `i` in the above example gives the insert mode cursors.

**Comment by gabesoft** [comment]
I'm not able to reproduce this. Just to be clear, so when you press `i` the cursor changes but not when you press `ci}`? What about the real cursor, does it change in both cases?

**Comment by gdkrmr** [comment]
yes, this is exactly the observed behavior, all cursors behave identically. I am using spacemacs on a very recent develop, maybe this makes the difference?
